pkg/sysinfo/numMem.go

```
@@ -0,0 +1,31 @@
+// +build amd64, arm, freebsd
+
+package sysinfo
+
+import (
+	"unsafe"
+
+	"golang.org/x/sys/unix"
+)
+
+// NumMem queries the system for the count of Memory Nodes available
+// for use to this process.
+func NumMem() int {
```

Suggest renaming to "NUMANodeCount". Also, should probably return an error.